doc/examples/applications/plot_pixel_graphs.py

```diff
@@ -35,7 +35,7 @@
 _, ax = plt.subplots()
 ax.imshow(retina_source)
 ax.set_axis_off()
-_ = ax.set_title('Human retina')
+ax.set_title('Human retina')
```

Comment:
I'd revert these kind of changes. The `_` is used to hide the object returned by `ax.set_title` which is unnecessarily shown in the gallery example otherwise.

Reply:
Ok, thanks for explaining.

Reply:
@lagru I like using `;` for hiding returned objects in Matplotlib, but Sphinx-gallery errors on them :cry:

Reply:
Reverted the changes, including ax instead of axes.